docs/src/pages/nextjs/appdir.mdx

```diff
@@ -53,12 +53,6 @@ export type OurFileRouter = typeof ourFileRouter;
 
 ### Create a Next.js API route using the FileRouter
 
-<Callout type="warning">
```

Comment:
Maybe change this to an `info` block -- we recommend placing this at `/api/uploadthing`...